client/styles/importer.less

```diff
@@ -1,3 +1,5 @@
+@cache-version: `process.env.npm_package_version` ;
```

Comment:
Yeah, I totally understand that, and I would be happy if we found a nicer solution, but the one you found really just rubs me the wrong way. I mean, the thing is that if you run without through npm, you won't get any errors either, you'll just get a version of "undefined", which is fine because you are doing stupid things (you have to go through a lot of steps to be able to run the command manually) then you get stupid things. We have the supported way of doing things, and that's the only way we can guarantee we will get what we need. I think that's perfectly fine. Anyway, I've reverted your commit, can you approve if you are happy.